codes/quantum/properties/lego.yml

```diff
@@ -0,0 +1,37 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: lego
+
+name: 'LEGO code'
+introduced: '\cite{arXiv:2109.08158}'
+
+description: |
+  The quantum LEGO method is a method of combining multiple codes that have been converted into Tensors by tying together the input and outputs of those codes, in a way stacking them similar to LEGOs.
+
+
+
+features:
+
+  encoders:
+    - 'First each code mapping \(V\) is converted to a tensor by decomposing it using the formula \(V = \sum_{i_j} V_{i_1 \ldots i_{n+k}} | i_{k+1} \ldots i_{k+n} \rangle \langle i_1 \ldots i_k |\). '
+    - 'We then look at the codes graphically, treating each \(i_j\) as an edge dangling out of the tensor vertex \(V_{i_1 \ldots i_{n+k}}\). These edges are either connected to another tensor vertex''s edges or left dangling.'
+    - 'Each local tensor has unitary product stabilizers (UPS). The goal is to push each UPS through the tensor network until each dangling edge has only trivial support. Otherwise, a matching value is pushed through the edge and the process is repeated on the next tensor. If a UPS can be pushed through the whole network, then a UPS for the larger network has been found.'
+    - 'The dangling legs (edges) and UPS of the whole network can then be converted to physical/logical elements and stabilizers/logical operators for a new quantum ECC.'
+
+  decoders:
+    - 'The decoder is created by creating a decoding quantum circuit with dangling legs replaced with input/output wires, and tensors converted to unitary gates.'
+    - 'Maximum likelihood decoding can be used when the tensors are stabilizer codes.'
+
+notes:
+  - 'The bounds for these codes depend entirely on the codes used to create them.'
+  - 'These codes can serve as a bridge between quantum error correction and physics. For example, the introduction paper states that finding the tensor constructions of well-known gauge codes using the lego method can help with the understanding of low energy Hamiltonian behavior with frustration in the ground space.'
```

Comment:
If there is no quantitative support for how the method "can help" then we can leave this out